pkg/txn/client/operator.go

```diff
@@ -505,7 +505,12 @@ func (tc *txnOperator) handleErrorResponse(resp txn.TxnResponse) error {
 	return nil
 }
 
-func (tc *txnOperator) checkResponseTxnStatusForReadWrite(txnMeta *txn.TxnMeta) error {
+func (tc *txnOperator) checkResponseTxnStatusForReadWrite(resp txn.TxnResponse) error {
+	if resp.TxnError != nil {
```

Comment:
So this is != nil, but we will return nil.    We ate this error and continue?

Reply:
handle in tc.checkTxnError

Reply:
checkResponseTxnStatus used to check if the status of TxnMeta returned by the server is correct